src/index.js

```diff
@@ -8,6 +8,10 @@ module.exports = function createServePlaceholder (_options) {
   const options = defu(_options, defaults)
 
   return function servePlaceholderMiddleware (req, res, next) {
+    // If response already sent, skip
+    if (res.headersSent) {
```

Comment:
Would be better using res.writableEnabled (https://nodejs.org/api/http.html#http_response_writableended)

Reply:
I can but this will reduce support to v12.9 but I guess it's fine since Node LTS is 14

Reply:
Thanks. Main reason was h3 is also working with newer flag.